src/System.Private.CoreLib/shared/System/Buffer.cs

```diff
@@ -241,21 +243,38 @@ internal static unsafe void Memmove(byte* dest, byte* src, nuint len)
             *(short*)(destEnd - 2) = *(short*)(srcEnd - 2);
             return;
 
-            MCPY05:
+        MCPY05:
             // PInvoke to the native version when the copy length exceeds the threshold.
+#if HAS_CUSTOM_BLOCKS
+            if (Avx.IsSupported)
```

Comment:
Is this going to be less efficient for ARM64, where it previously would have gone down the `Block64` path and been able to use 4 sequential 16-byte moves (and now will be doing a single 16-byte move at a time)?

Reply:
Deleted this bit; but the .IsSupported check was only used to increase the pinvoke threshold